pennylane/init.py

```diff
@@ -1127,3 +1127,69 @@ def basic_entangler_layers_uniform(n_layers, n_wires, low=0, high=2 * pi, seed=N
     params = np.random.uniform(low=low, high=high, size=(n_layers, n_wires))
 
     return params
+
+
+def gate_fabric_uniform(n_layers, n_wires, low=0, high=2 * pi, seed=None):
```

Comment:
oh @obliviateandsurrender we haven't done a very good job of documenting it, but the `init` module is deprecated! Instead of adding new functions here, the preferred approach is to have a `shape` method on the template itself:

```pycon
>>> shape = qml.templates.StronglyEntanglingLayers.shape(n_layers=3, n_wires=2)
>>> np.random.random(shape)
array([[[0.48715978, 0.24741362, 0.19435542],
        [0.54890558, 0.58210102, 0.98678809]],

       [[0.43263127, 0.09268041, 0.4359657 ],
        [0.34308469, 0.79198242, 0.81406353]],

       [[0.80658304, 0.45946463, 0.66767815],
        [0.75200523, 0.46483252, 0.02140596]]])
```

This has the advantage that we don't need to wrap NumPy's random module, worry about seeds, etc.

Also tagging @antalszava and @mariaschuld here for visibility; we should decide if we remove/fully deprecate this module.

Reply:
Hey @josh146! I am sorry I did not know about `init` module's depreciation. Anyhow, I already had included the `shape` method in `GateFabric` template. I have now removed the previously added functions from the `init`.

Reply:
No worries at all, the PennyLane documentation was not clear as to the status of the init module 🙂  This is something we should address

Reply:
Maybe for now we can just add a note in the docstring and/or remove it from the documentation and all tutorials? So it would still work, but people won't find it or add to it?

Reply:
:+1: Let's also add it to one of our upcoming deprecation cycles @antalszava, so that we don't forget about it

Reply:
I started a PR [here](https://github.com/PennyLaneAI/pennylane/pull/1689/files)...

Reply:
Just to summarize, would the following sound good?
* v0.19.0 (next release):
    * The `init` module is still available, but not advertised in the docs and a deprecation warning has been added as per https://github.com/PennyLaneAI/pennylane/pull/1689
    * Demos don't use the `init` module anymore
* v0.20.0: the `init` module is completely removed from PennyLane

Reply:
That sounds good to me :+1;